src/commands/path/path.generate.ts

```diff
@@ -44,6 +44,14 @@ export const commandGeneratePath = command({
       description: 'Target bucket name, e.g. nz-imagery',
     }),
 
+    ignoreDate: flag({
```

Comment:
"Ignore" sounds a bit vague to me. How about `--no-include-date` or even `--no-date-in-survey-name`?

Reply:
`--exclude-date`?

Reply:
Maybe? It's kind of confusing as well, since it sounds like an option to exclude a certain date from the dataset. What do others think?

Reply:
`--no-date-in-survey-path`?

Reply:
`--no-date-in-survey-path` is nice and specific